src/main/kotlin/step3_simple_racing_car/io/GameOutput.kt

```diff
@@ -0,0 +1,15 @@
+package step3_simple_racing_car.io
+
+import step3_simple_racing_car.domain.RacingCar
+
+object GameOutput {
+    fun printGameResult(
+        participants: List<RacingCar>,
+    ) {
+        participants.forEach {
+            (1..it.position.value).map { _ -> print("-") }
+            println()
```

Comment:
```suggestion
            println("-".repeat(it.position.value))
```
제꺼 미션에서 쓴 코드 공유합니다. ㅎㅎ

Reply:
오이거 알고리즘때 쓰던 습관이..ㅋㅋㅋ 다른건 다 고쳤는데 이부분만 repeat()으로 바꾸는걸 누락했군요..!!
경현님 추천코드 복붙 갑니다 ㅎㅎ 감사합니다!